var/spack/repos/builtin/packages/googletest/package.py

```diff
@@ -67,5 +71,9 @@ def install(self, spec, prefix):
                          prefix.include)
 
             mkdirp(prefix.lib)
-            install('libgtest.a', prefix.lib)
-            install('libgtest_main.a', prefix.lib)
+            if '+shared' in spec:
+                install('libgtest.{0}'.format(dso_suffix), prefix.lib)
+                install('libgtest_main.{0}'.format(dso_suffix), prefix.lib)
+            else:
+                install('libgtest.a', prefix.lib)
+                install('libgtest_main.a', prefix.lib)
```

Comment:
What versions of this package have these changes been tested for???  Because the changes to the install procedure don't even take effect for the latest version.

Reply:
@citibeth, as you note the changes I'm working on for you only apply to versions up to 1.7.0.  This is because the latest, 1.8.0, has an install target and knows internally how to install shared/static libs.  I have tested this PR against all versions in shared/non-shared configurations with success on Linux.  I do not have a MacOS test platform so I can't speak in that arena.  If you and @adamjstewart want me to commit a follow-on to set +shared as the default, I'm glad to.  I was just keeping the same default as before so that folks using it would not be surprised by the change.  Please advise ...

Reply:
I'm happy with the testing you've done.  I'm happy to forgo macOS testing, you've followed the pattern that generally works.

If it were up to me, I'd lean toward leaving the `shared` default as-is.  But @adamjstewart will ultimately merge this PR, so he's the one you have to convince.